bpf/cpu/cpu.bpf.c

```diff
@@ -55,6 +55,9 @@ _Static_assert(1 << MAX_BINARY_SEARCH_DEPTH >= MAX_UNWIND_TABLE_SIZE, "unwind ta
 // Special values.
 #define CFA_TYPE_END_OF_FDE_MARKER 4
 
+#define CFA_TYPE_FP_ARM 5
+#define CFA_TYPE_SP_ARM 6
```

Comment:
As discussed offline, let's try to unify these. If in the future we realise that for arm64 we need some different fields / ids, we can always change it back.

The benefit of keeping it unified is that is a bit more clear and elegant perhaps, but also that it will reduce the amount of code we'll need in the BPF programs. While it won't likely be a significant increase, every instruction we do not run matters